packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -980,15 +980,7 @@ export class LitNodeClientNodeJs
    *
    */
   getSignature = async (shareData: any[], requestId: string): Promise<any> => {
```

Comment:
As per our discussion in slack, this can be removed too. Just that we need to remove it from the `ILitNodeClient` interface too
Or we can deprecate it with depd if we see it makes sense to hang to this until next breaking change